kubeflow/jupyter/jupyter-web-app.libsonnet

```diff
@@ -87,93 +87,32 @@
         apiGroup: "rbac.authorization.k8s.io",
       },
     },
-    
-    notebookRole:: {
-      apiVersion: "rbac.authorization.k8s.io/v1beta1",
-      kind: "Role",
-      metadata: {
-        name: "jupyter-notebook-role",
-        namespace: params.namespace,
-      },
-      rules: [
-        {
-          apiGroups: [
-            "",
-          ],
-          resources: [
-            "pods",
-            "pods/log",
-            "secrets",
-            "services",
-          ],
-          verbs: [
-            "*",
-          ],
-        },
-        {
-          apiGroups: [
-            "",
-            "apps",
-            "extensions",
-          ],
-          resources: [
-            "deployments",
-            "replicasets",
-          ],
-          verbs: [
-            "*",
-          ],
-        },
-        {
-          apiGroups: [
-            "kubeflow.org",
-          ],
-          resources: [
-            "*",
-          ],
-          verbs: [
-            "*",
-          ],
-        },
-        {
-          apiGroups: [
-            "batch",
-          ],
-          resources: [
-            "jobs",
-          ],
-          verbs: [
-            "*",
-          ],
-        },
-      ],
-    },
-    
-    notebookServiceAccount:: {
+
+    defaultEditorServiceAccount:: {
       apiVersion: "v1",
       kind: "ServiceAccount",
       metadata: {
-        name: "jupyter-notebook",
+        name: "default-editor",
```

Comment:
Is this service account going to be shared between different components?
If so, it should not be part of jupyter-webapp right?

Reply:
Agree, that would be the final state.
As of today since we start by moving only notebooks to the new auth model, shall we take a shortcut here?

Reply:
I see.
Can you file an issue and add TODO here to move this sa and rolebinding ?